docs(react-core): add note about combining plugins (#2677)
DevExpress/devextreme-reactive: packages/dx-react-grid/docs/guides/plugin-overview.md

> The data processing plugins' order is also important because they transform data in the same order they appear. For example, if the `IntegratedPaging` plugin precedes the `IntegratedSelection`, the 'Select All' checkbox selects only the current page's rows and swapping them around allow selecting rows on all pages. See this rule in action in the following demo:

Reviewer: "they transform data in the same order they appear" Why 'transform'?

Reviewer 2: I guess "transform" was used as a synonym of the word "process". It makes sense to use "process" because the plugins are data _processing_. How do you think?

Reviewer: ```suggestion
The data processing plugins' order is also important because they process data in the same order they appear. For example, if `IntegratedPaging` precedes `IntegratedSelection`, the 'Select All' checkbox selects only the current page's rows and swapping them around allows you to select rows on all pages. See this rule in action in the following demo:
```

.embedded-demo({ "path": "grid-selection/select-all-by-page", "showThemeSelector": true })

## Combining Multiple Plugins

> Use a Plugin component to wrap multiple plugins into a single component as demonstrated in the sample below.

Reviewer 2: ```suggestion
Use the `Plugin` component to wrap multiple plugins into a single component:
```
